### ISSUE TITLE
click on `untagged stars` should exclude automatic language stars

### ISSUE THREAD

**Issue by muescha** [astralapp/astral#119]
click on `untagged stars` should exclude automatic language stars

**Comment by syropian** [comment]
I don't agree, and it would unexpected behaviour for most users.

**Comment by muescha** [comment]
if i see over the first text, then it can be confusing. i would like to explain my use case in detail.

use case:
i am new to astral

default was 'automatic language' set OFF

i find the `untagged stars` button useful, because now i can work though my repos and add tags. i have so much (1.8k), so i can come back later, hit on `untagged stars` and continue the tagging

i would not only tag with language but also with my use case or with my special app where i can use a repo

then i switched on the `automatic language` tagging

after reload the page all my repos got a atomatic language tag


if i now hit on `untagged stars` i get only a few (about 30) of my starred repos

now i try to switch off the automatic tagging - but the language tags are still there 

and now i dont have a chance to go through my stars where i dont have done a manual tag



PS:
or call it `custom untagged stars` or `user untagged stars` 

or: if i switch off the `automatic language tagging` then a click on `untagged stars` should display the stars where i dont have added a custom tag (and ignore for the filter untagged the language tags)

**Comment by syropian** [comment]
Since languages tags are no different than any other tags, it's impractical to make that change at this time. It would be different if we treated language tags differently, but since we don't I'm not comfortable making a UX change like this.

I have played around with the idea of separating languages from tags, but still making languages part of the filtering equation, but at this point it would be tough to migrate all the current user data to a new system.

I think once I add more advanced filtering, which can provide options like `exclude language autotags` this will be possible, but I'm not up for changing the functinality of the `Untagged` filter at this time.
